tests/e2e/README.md

```diff
@@ -0,0 +1,25 @@
+# End-to-End Testing
+
+## Context
+
+The FSx OpenZFS CSI driver utilizes a variety of workflows to validate end to end functionality in a Kubernetes environment.
+The framework and existing testsuites for Kubernetes storage E2E testing is leveraged to test common cases.
+Untested or unique test cases are also tested using a custom testsuite.
+
+Due to the nature end-to-end tests it is possible to leave testing artifacts stranded.
+Events such as abrupt termination or server-side API failures can prevent resources from being deleted properly.
+The `fsx_delete_e2e_resources` hack cleans up all testing resources that may have been left stranded.
```

Comment:
I don't see this bash script in this commit, are you still debugging it?

Reply:
Oops good catch. I forgot to add the bash script into the commit. It's good to go.